internal/common/shared/iigocommunications.go

```diff
@@ -71,6 +71,8 @@ const (
 	SanctionAmount
 	RoleMonitored
 	MonitoringResult
+	IIGOSanctionTier
+	IIGOSanctionScore
```

Comment:
did you update the String() function

Reply:
Crap, just realised this Enum hasn't been moved over to that model. Let me add that.

Reply:
Nvm I was confused, added to the .String() function